[2025-01-31T04:44:21.830+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2021-12-03T00:00:00+00:00 [queued]>
[2025-01-31T04:44:21.844+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2021-12-03T00:00:00+00:00 [queued]>
[2025-01-31T04:44:21.845+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T04:44:21.906+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2021-12-03 00:00:00+00:00
[2025-01-31T04:44:21.915+0000] {standard_task_runner.py:57} INFO - Started process 2690 to run task
[2025-01-31T04:44:21.919+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v2', 'second_task', 'scheduled__2021-12-03T00:00:00+00:00', '--job-id', '654', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmp64w9ptkj']
[2025-01-31T04:44:21.921+0000] {standard_task_runner.py:85} INFO - Job 654: Subtask second_task
[2025-01-31T04:44:22.009+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v2.second_task scheduled__2021-12-03T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T04:44:22.126+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-12-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-03T00:00:00+00:00'
[2025-01-31T04:44:22.127+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T04:44:22.128+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the second task, I run after the first task']
[2025-01-31T04:44:22.142+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T04:44:22.144+0000] {subprocess.py:93} INFO - I am the second task, I run after the first task
[2025-01-31T04:44:22.145+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T04:44:22.176+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v2, task_id=second_task, execution_date=20211203T000000, start_date=20250131T044421, end_date=20250131T044422
[2025-01-31T04:44:22.217+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T04:44:22.241+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
